api/src/Controller/StatsController.php

```diff
@@ -78,4 +78,68 @@ public function getAdminUserAccountReportData(Request $request, RestFormatter $f
             'ActivatedAdminAccountsUsedWithin90Days' => $countOfActivatedAdminAccountsUsedWithin90Days,
         ];
     }
+
+    /**
+     * @Route("stats/assets/total_values", methods={"GET"})
+     * @Security("is_granted('ROLE_SUPER_ADMIN')")
+     */
+    public function getAssetsTotalValueData()
+    {
+        $ret = [
+            'lays' => ['liquid' => 0, 'non-liquid' => 0],
+            'profs' => ['liquid' => 0, 'non-liquid' => 0],
+            'pas' => ['liquid' => 0, 'non-liquid' => 0],
+            'grandTotal' => 0,
+        ];
+
+        $lays = $this->reportRepository->getAllSubmittedReportsWithin12Months('LAY');
+        $profs = $this->reportRepository->getAllSubmittedReportsWithin12Months('PROF');
+        $pas = $this->reportRepository->getAllSubmittedReportsWithin12Months('PA');
+        $ndrs = $this->ndrRepository->getAllSubmittedNdrsWithin12Months();
+
+        $layClientIds = [];
+
+        foreach ($lays as $layReport) {
```

Comment:
Given the size of these chunks, worth checking how much memory this takes up in PHP. Might be better pushed down to SQL.

Reply:
I was trying to avoid the complexity that's going to be involved with handling the different report types and then excluding NDRs for reports that we've already counted. 

I've taken a look in preprod and in the last 12 months we have 32K~ submitted reports so shouldn't have any systemwide impact querying that many rows. I'd expect PHP to be able to handle chunks of that size without running out of memory but will run on preprod to be sure before releasing though. 

Given the short notice for the request (Monday) I've not added tests for large data sets which I hate but just trying to prioritise against the CSV work.

Reply:
Interesting...will see what happens on preprod 🤞